docs/api.md

```diff
@@ -1331,6 +1331,17 @@ yargs.showHelp("log"); //prints to stdout using console.log()
 
 Later on, `argv` can be retrieved with `yargs.argv`.
 
+.showHelp(printCallback)
+---------------------------
+
+Print the usage data using the specified callback, which accepts a single argument - the string to print.
```

Comment:
If I recall, can `showHelp` take no callback and it will result in help printing to `stdout`?

Reply:
@bcoe Right. Added doc section (and test) to explicitly describe that.